### PR TITLE
Kindly Review Mine: faisal.json

### PR DESCRIPTION
**Pull request by faisal-fida**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: faisal-fida.github.io
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

**Comment by fasih4444** [comment]
@tonetheman @frozzare @filipekiss @stupergenius Kindly Review.

**Comment by VaibhavSys** [comment]
> @tonetheman @frozzare @filipekiss @stupergenius Kindly Review.

Don't ping random people for review.
They aren't even maintainers.

**Comment by fasih4444** [comment]
@VaibhavSys Oh Sorry About that.

File: domains/faisal.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "faisal's portfolio website",
+  "repo": "https://github.com/fasih4444/Portfolio-V4",
+  "owner": {
+    "username": "fasih4444",
+    "email": "arainfaisal826@gmail.com",
+    "twitter": "fasih4444"
+  },
+  "record": {
+    "CNAME": "faisal.vercel.app"
```
Comment (VaibhavSys) [inline domains/faisal.json:10]: Vercel isn't supported.
It was clearly stated in the check list that Vercel isn't supported.
Try another service like GitHub Pages or [Railway](https://railway.app)

**Review by VaibhavSys — CHANGES_REQUESTED** [review]

File: domains/faisal.json (hunk 1 above)
Comment (fasih4444) [reply domains/faisal.json:10]: OK Vaibhav, I'll ping you after update.

Comment (faisal-fida) [reply domains/faisal.json]: > Vercel isn't supported. It was clearly stated in the check list that Vercel isn't supported. Try another service like GitHub Pages or [Railway](https://railway.app)

@VaibhavSys Updated Github.io URL. Kindly Review.

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**